processors/tenantidprocessor/facotry.go

```diff
@@ -0,0 +1,60 @@
+package tenantidprocessor
+
+import (
+	"context"
+	"go.opencensus.io/stats/view"
```

Comment:
Do we truly need this?

Reply:
if we want to use metrics then yes. I would stick with metrics API that the upstream uses, there is plan to migrate but who knows when https://github.com/open-telemetry/opentelemetry-collector/issues/816